### PR TITLE
chore: sync with runtime

### PR DESCRIPTION
**Pull request by jh2oman**
### Description

The following files had conflicts manually:
```
	both modified:   src/components/ContactCircle.tsx
	both modified:   src/components/SelectRecipientButton.tsx
	both modified:   src/components/__snapshots__/Avatar.test.tsx.snap
	both modified:   src/components/__snapshots__/ContactCircle.test.tsx.snap
	both modified:   src/recipients/RecipientItemV2.tsx
	both modified:   src/send/__snapshots__/SendConfirmation.test.tsx.snap
	deleted by them: src/transactions/__snapshots__/NoActivity.test.tsx.snap
```

### Test plan

Conflicts had to do with send flow. manually verified that styles match the new send flow changes made in https://github.com/valora-inc/wallet/pull/6220.

### CODE REVIEW

File: src/components/SelectRecipientButton.tsx [hunk 1]
```diff
@@ -22,16 +24,20 @@ function SelectRecipientButton({
...
   return (
     <Touchable testID={testID} onPress={onPress} style={styles.container}>
       <View style={styles.body}>
-        <CircledIcon radius={40} borderColor={colors.black} backgroundColor={iconBackgroundColor}>
-          {icon}
-        </CircledIcon>
+        {gradientBackground ? (
+          <GradientIcon radius={40}>{icon}</GradientIcon>
+        ) : (
+          <CircledIcon radius={40} style={styles.icon} backgroundColor={colors.gray1}>
```
Comment (satish-ravi) [inline src/components/SelectRecipientButton.tsx]: I think this should be white to match the new designs?

File: src/recipients/RecipientItemV2.tsx [hunk 2]
```diff
@@ -57,8 +59,8 @@ function RecipientItem({ recipient, onSelectRecipient, loading, selected }: Prop
             recipient={recipient}
             backgroundColor={Colors.white}
             foregroundColor={Colors.black}
-            borderColor={Colors.black}
-            DefaultIcon={() => <QuestionIcon />} // no need to honor color props here since the color we need match the defaults
+            borderColor={Colors.gray2}
```
Comment (satish-ravi) [inline src/recipients/RecipientItemV2.tsx]: black to match the new designs?

**Comment by satish-ravi** [comment]
consider renaming the commits you made on top, so that when we do a "merge commit", conventional commit style is retained

**Comment by jophish** [comment]
@jh2oman i think you may have lost all the individual commits somehow? i'm no longer seeing them in the history

**Review by MuckT — DISMISSED** [review]

**Comment by jh2oman** [comment]
Yea i messed up :) re-doing now.

**Review by jophish — APPROVED** [review]

**Review by jophish — COMMENTED** [review]
for some reason, it looks like [your commit](https://github.com/mobilestack-xyz/mobilestack-mento/pull/26/commits/b8faa4d5bd86d34267b1b7009043e08d7913b4f3) contains _all_ the previous changes, not just the merge conflict resolution... this seems, strange. @satish-ravi did we run into this when doing the sync last week?

**Comment by satish-ravi** [comment]
@jophish I think that might be expected since thats the "merge" commit. I see the same here https://github.com/mobilestack-xyz/mobilestack-runtime/commit/9c3aa5051faaf87969e385e745b9f7fd31ae1234

**Review by satish-ravi — APPROVED** [review]

**Pull request merged**